.vib/drupal/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,29 @@
+const COMMAND_DELAY = 2000;
+
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, COMMAND_DELAY);
+    });
+  });
+}
+
+Cypress.Commands.add(
+  'login',
+  (username = Cypress.env('username'), password = Cypress.env('password')) => {
+    cy.visit('/user/login');
+    cy.get('#edit-name').type(username);
+    cy.get('#edit-pass').type(password);
+    cy.contains('input', 'Log in').click();
+  }
+);
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  if (err.message.includes('Unexpected token')) {
+    return false;
+  }
+});
```

Comment:
Is this exception handling necessary? It seems strange to me that drupal outputs an `Unexpected token` error.

Reply:
I've removed this now.